civihr_default_theme/scss/components/_header.scss

```diff
@@ -218,6 +218,19 @@
   }
 }
 
+@media (max-width: $screen-md-max - 1) {
```

Comment:
This also does not sound right to me. If let's say `$screen-md-max === 500`, then it means that `500` would still be **screen-md**. In this rule you say `for screen that is less or equal 499`.

Am I missing something?

Also, not sure why do you need this rule, you can simply position the *default* rules and then override with media query below. You don't need two media queries.

Reply:
After correcting all variables in the file as you mentioned, `-1` is not necessary anymore.

> Also, not sure why do you need this rule, you can simply position the default rules and then override with media query below. You don't need two media queries.

Done